tests/span_tests_qc.rs

```diff
@@ -0,0 +1,233 @@
+use quickcheck::Arbitrary;
+use tabled::{builder::Builder, object::Cell, Modify, Span, Style};
+
+#[derive(Clone, Debug)]
+struct TableStructure {
+    pub rows: Vec<Line>,
+    pub theme: ThemeFixture,
+    pub row_span: Vec<u8>,
+    pub col_span: Vec<u8>,
+}
+
+type Line = Vec<u32>;
```

Comment:
(a note)

Maybe it's even better then `Vec<String>`.
From speed perspective at least but I don't know.